src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,87 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private List<String> listOfFilesData = new ArrayList<>();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder dataToWrite = new StringBuilder();
+        readDataFromFile(fromFileName);
```

Comment:
```suggestion
        List<String> dataFromFile = readDataFromFile(fromFileName);
```

Reply:
To assign the value of this method to a field, you must change the return type of the readDataFromFile method, which makes no sense

Reply:
it makes sense